.github/workflows/clabbert.yml

```diff
@@ -0,0 +1,55 @@
+name: clabbert-ci
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "implemntation/clabbert/**"
+      - ".github/workflows/clabbert.yml"
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - ready_for_review
+    branches:
+      - main
+    paths:
+      - "implemntation/clabbert/**"
+      - ".github/workflows/clabbert.yml"
+
+jobs:
+  build:
+    if: github.event.pull_request.draft == false
+    name: Build and test project
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    defaults:
+      run:
+        working-directory: implementation/clabbert
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
**question:** Why do we need earlier commits?

Reply:
My bad, this is the result of not understanding and misinterpreting what fellow copilot has produced.